core/src/test/java/org/apache/iceberg/avro/TestAvroOptionsWithNonNullDefaults.java

```diff
@@ -40,6 +41,9 @@
 
 public class TestAvroOptionsWithNonNullDefaults {
 
+  String fieldWithDefaultName = "fieldWithDefault";
```

Comment:
Maybe `private static final` for consistency here and other places?

Reply:
done